### DISCUSSION TITLE
Show timeout value when latency is too high instead of "0 ms"

### DISCUSSION THREAD

**Discussion by MericGit** [BattlesnakeOfficial/feedback#53] [Feature Requests]
Currently when your latency is over the 500 limit it displays as 0ms
![image](https://user-images.githubusercontent.com/41242144/119245731-02d95d00-bb41-11eb-88d9-e658da8672d4.png)

Changing it to display "500 ms" instead would allow for more clarity and reduce some confusion.

**Comment by andrewdmaclean** [comment]
Thanks for the suggestion! 

We will make sure it gets passed onto the Engineering team.

**Reply by andrewdmaclean** [reply]
Hey @MericGit just wanted to let you know that we are looking into this so you didn't think it was just left.

**Comment by bvanvugt** [comment]
Yeah, we're going to fix this. Created Issue #67 to track.

**Comment by robbles** [comment]
This bug has now been fixed - thanks for reporting it!
